IES Specification Docs/ont.file.ttl

```diff
@@ -0,0 +1,3919 @@
+
+ies:governedRegionRangeShape a sh:NodeShape ;
+    sh:class ies:Location ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:governedRegion .
+
+ies:grantedActivityTypeDomainShape a sh:NodeShape ;
+    sh:class ies:GrantOfAuthority ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:grantedActivityType .
+
+ies:grantedActivityTypeRangeShape a sh:NodeShape ;
+    sh:class ies:AuthorisedEventClass ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:grantedActivityType .
+
+ies:groupDescriptionDomainShape a sh:NodeShape ;
+    sh:class ies:GroupOfItems ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:groupDescription .
+
+ies:groupNameDomainShape a sh:NodeShape ;
+    sh:class ies:GroupOfItems ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:groupName .
+
+ies:handlingCaveatDomainShape a sh:NodeShape ;
+    sh:class ies:SecurityLabel ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:handlingCaveat .
+
+ies:happensInDomainShape a sh:NodeShape ;
+    sh:class ies:Event ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:happensIn .
+
+ies:happensInRangeShape a sh:NodeShape ;
+    sh:class ies:Location ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:happensIn .
+
+ies:hasAccessToDomainShape a sh:NodeShape ;
+    sh:class ies:ResponsibleActorState ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:hasAccessTo .
+
+ies:hasAccessToRangeShape a sh:NodeShape ;
+    sh:class ies:Entity ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:hasAccessTo .
+
+ies:hasAuthorDomainShape a sh:NodeShape ;
+    sh:class ies:WorkOfDocumentation ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:hasAuthor .
+
+ies:hasAuthorRangeShape a sh:NodeShape ;
+    sh:class ies:ResponsibleActor ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:hasAuthor .
+
+ies:hasCharacteristicDomainShape a sh:NodeShape ;
+    sh:class ies:Element ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:hasCharacteristic .
+
+ies:hasCharacteristicRangeShape a sh:NodeShape ;
+    sh:class ies:Characteristic ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:hasCharacteristic .
+
+ies:hasCountryOfIssueDomainShape a sh:NodeShape ;
+    sh:class ies:IdentityDocument,
+        ies:TravelCard ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:hasCountryOfIssue .
+
+ies:hasCountryOfIssueRangeShape a sh:NodeShape ;
+    sh:class ies:Country ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:hasCountryOfIssue .
+
+ies:hasEmergencyContactAddressDomainShape a sh:NodeShape ;
+    sh:class ies:IdentityDocument ;
+    sh:severity sh:Warning ;
+    sh:targetSubjectsOf ies:hasEmergencyContactAddress .
+
+ies:hasEmergencyContactAddressRangeShape a sh:NodeShape ;
+    sh:class ies:Address ;
+    sh:severity sh:Warning ;
+    sh:targetObjectsOf ies:hasEmergencyContactAddress .
+
+ies:hasEthnicityDomainShape a sh:NodeShape ;
```

Comment:
all shape class localnames should be capitalised

Reply:
Done